package.json

```diff
@@ -47,10 +48,10 @@
     "react-dom": "^17.0.2",
     "react-ga": "^2.7.0",
     "react-helmet": "^5.2.1",
-    "react-markdown": "^4.2.2",
+    "react-markdown": "^8.0.3",
     "react-redux": "^7.1.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5",
```

Comment:
I don't recommend upgrading `create-react-app` to version 5.

5.0.1 is still not stable for my taste. Plus, opendota has a lot of old dependencies, something may go wrong.